go/libkb/util_test.go

```diff
@@ -41,3 +41,10 @@ func TestCombineErrors(t *testing.T) {
 		t.Errorf("Wrong output for combine errors: %#v != %#v", err.Error(), expected)
 	}
 }
+
+func TestWriteVersion(t *testing.T) {
+	err := WriteVersion("1.2.3", 400, "")
+	if err != nil {
+		t.Errorf("%s", err)
+	}
```

Comment:
In a perfect world I'd love to test that `VersionString()` now reflects the new version file. One way to do that would be to shell out to `packaging/version.sh`, if we're not planning to delete that script.